recipe/meta.yaml

```diff
@@ -1,15 +1,13 @@
 {% set name = "pyepics" %}
-{% set version = "3.2.6" %}
-{% set sha256 = "630ac60bf9362a758211cfd3af4e62c9a0a394af6e5674aababdf6c857ad34a5" %}
+{% set version = "3.3.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/pyepics/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_rev: {{ version }}
+  git_url: https://github.com/pyepics/pyepics.git
```

Comment:
Please pull from tarballs not git https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos